### DISCUSSION TITLE
Just installed ark for solid: some component didn’t resolve

### DISCUSSION THREAD

**Discussion by promethyttrium** [chakra-ui/ark#2474] [General]
### Description

![image](https://github.com/chakra-ui/ark/assets/3183017/98b12f26-9502-4c3d-a5a2-42308b82521a)

404 when trying to look at the docs: https://ark-ui.com/docs/components/menu

- Indicator does not resolve
- value does not exist

Why doesn't the example work?

### Link to Reproduction (or Detailed Explanation)

See steps below

### Steps to Reproduce

1. Followed the instructions at https://ark-ui.com/docs/solid/overview/getting-started
2. Copied the component for Menu:  (https://ark-ui.com/docs/solid/components/menu)
3. See screenshot 

### Ark UI Version

^2.2.0

### Framework

- [ ] React
- [X] Solid
- [ ] Vue

### Browser

Didn't get to the browser

**Comment by cschroeter** [comment]
@promethyttrium 

Please use `3.0.0-6` will file a new major shortly.
